api/test/opentelemetry/instrumentation/base_test.rb

```diff
@@ -298,14 +298,15 @@ def initialize(*args)
   end
 
   describe '#tracer' do
-    it 'returns nil if not installed' do
-      _(instrumentation_with_callbacks.instance.tracer).must_be_nil
+    it 'returns a noop api tracer if not installed' do
+      _(instrumentation_with_callbacks.instance.tracer).must_be_instance_of(OpenTelemetry::Trace::Tracer)
+      _(instrumentation_with_callbacks.instance.tracer).wont_equal(OpenTelemetry.tracer_provider.tracer)
```

Comment:
This and the check below seem overly specific. The previous check is sufficient.

Reply:
Fair, updated!